src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -286,27 +287,11 @@ ov::element::Type FullyConnected::getRuntimePrecision() const {
 }
 
 void FullyConnected::fuseDecompressionMultiply(const MemoryCPtr& memory) {
-    fuseDecompressionConstant(memory, attrs.decompressionMultiplyPtr);
+    attrs.decompressionMultiplyPtr = memory;
 }
 
 void FullyConnected::fuseDecompressionSubtract(const MemoryCPtr& memory) {
-    fuseDecompressionConstant(memory, attrs.decompressionSubtractPtr);
-}
-
-void FullyConnected::fuseDecompressionConstant(const MemoryCPtr& memory, MemoryCPtr& decompressionValuesPtr) {
-    const auto decompression_prc = ov::element::f32;
-    if (memory->getDesc().getPrecision() == decompression_prc) {
-        decompressionValuesPtr = memory;
-    } else {
-        DnnlBlockedMemoryDesc memoryDesc(decompression_prc, memory->getShape());
-        decompressionValuesPtr = std::make_shared<Memory>(getEngine(), memoryDesc, nullptr, false);
-        const auto elementsCount = memory->getDescWithType<BlockedMemoryDesc>()->getPaddedElementsCount();
-        cpu_convert(memory->getData(),
-                    decompressionValuesPtr->getData(),
-                    DnnlExtensionUtils::DataTypeToElementType(memory->getDataType()),
-                    ov::element::f32,
-                    elementsCount);
-    }
```

Comment:
Could you please elaborate why we remove this logic?

Reply:
Now precision conversion logic is combined with repacking processing. Both handled inside [Memory::load(...) call](https://github.com/openvinotoolkit/openvino/blob/70d762c90050523b94ff88a7615475027457f023/src/plugins/intel_cpu/src/dnnl_postops_composer.cpp#L613) which utilizes oneDNN reorder underneath.

Reply:
Thanks for the explanation